packages/liveblocks-react/src/compat.tsx

```diff
@@ -267,7 +267,7 @@ export function useList<TValue extends Lson>(
   deprecate(
     "Please use `createRoomContext()` instead of importing `useList` from `@liveblocks/react` directly. See https://liveblocks.io/docs/guides/upgrading#upgrading-from-0-16-to-0-17 for details."
   );
-  return _hooks.deprecated_useList(key as any, items as any);
+  return _hooks.useList_deprecated(key as any, items as any);
```

Comment:
Rules of Hooks™ enforce that hooks always start with `use`-prefix. So I'm flipping these names here.